CHANGELOG.md

```diff
@@ -13,6 +15,7 @@
 - Added the `io.improbable.gdk.debug` package as a dependency. [#222](https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/222)
 
 ### Changed
+- Upgraded to GDK for Unity version `0.2.6`
 
```

Comment:
The newline is a little funky but it renders correctly

Reply:
add it to the list

Reply:
Already done so!